manifests/setting.pp

```diff
@@ -38,8 +33,13 @@
   }
 
   if ($setting_type == 'list') or ($setting_type == 'pref') {
-    $_priority = ''
+    $_priority = undef
```

Comment:
Could you make this continue to be `''`? The string interpolation below will always happen so empty string is cleaner.